### ISSUE TITLE
False positive for sentry/no-dynamic-translations

### ISSUE THREAD

**Issue by dashed** [getsentry/eslint-config-sentry#210]
`tct('[foo]' + 'bar', {foo: 'baz'})` generates a false positive: 

```
error  tct() cannot be used to translate dynamic values. Use a parameterized string literal instead  sentry/no-dynamic-translations
```

Use cases to split strings like this is for big and long copies to make them readable over multiple lines.
